pywbem_mock/_baseprovider.py

```diff
@@ -88,6 +76,16 @@ def __init__(self, cimrepository=None, provider_registry=None):
             repository is fully initialized.
 
           provider_registry (dict): Provider registry.
```

Comment:
The only derived class that passes the `provider_registry` init parameter to this class or uses it afterwards is `ProviderDispatcher`, and that makes sense. But that means `provider_registry` should be an attribute at the level of the `ProviderDispatcher` class. I suggest to remove the `provider_registry` init parameter from this class here and to move the initialization of the corresponding instance attribute to `ProviderDispatcher`.

Reply:
DONE

Reply:
It should also be removed from the docstring.

Reply:
DONE

Reply:
DONE